fortitude_dev/src/main.rs

```diff
@@ -0,0 +1,51 @@
+//! This crate implements an internal CLI for developers of Fortitude.
+//!
+//! Within the fortitude repository you can run it with `cargo dev`.
```

Comment:
I couldn't get this to work, but going into `fortitude_dev` and using `cargo run` was fine. Please could we update this here?

Reply:
`cargo dev` is working now, thanks!